packages/KAlert/KAlert.vue

```diff
@@ -4,77 +4,110 @@
     :class="[
       appearance,
       size,
-      type,
       dismissType,
-      {'isBordered':isBordered},
-      {'hasLeftBorder':hasLeftBorder},
-      {'hasRightBorder':hasRightBorder},
-      {'hasTopBorder':hasTopBorder},
-      {'hasBottomBorder':hasBottomBorder},
-      {'isCentered': isCentered},
-      {'isFixed': isFixed}
+      { 'is-bordered': isBordered },
+      { 'has-left-border': hasLeftBorder },
+      { 'has-right-border': hasRightBorder },
+      { 'has-top-border': hasTopBorder },
+      { 'has-bottom-border': hasBottomBorder },
+      { 'is-centered': isCentered },
+      { 'is-fixed': isFixed },
+      { 'is-banner': type === 'banner' }
     ]"
     class="k-alert"
     role="alert"
-    @click.stop>
+    @click.stop
+  >
+    <span
+      v-if="type === 'banner' && size !== 'large'"
+      :class="appearance"
+      class="k-alert-ellipse"
+    />
+    <span
+      v-if="icon || $slots.icon"
+      :class="{
+        'mr-3': type !== 'banner',
+        'k-alert-icon-container-large': size === 'large'
+      }"
+      class="k-alert-icon-container"
+    >
+      <slot name="icon">
+        <KIcon
+          :size="iconSize"
+          :color="iconColor"
+          :icon="icon"
+          class="k-alert-icon"
+        />
+      </slot>
+    </span>
+    <div class="k-alert-msg-text">
+      <div
+        v-if="title || $slots.title"
+        class="k-alert-title bold-600"
+      >
+        <slot name="title">
+          {{ title }}
+        </slot>
+      </div>
+      <div
+        :class="{
+          'k-alert-text': size === 'large',
+          'k-alert-subtext': title || $slots.title
+        }"
```

Comment:
Non-blocking question: is it expected and ok to have cases where both of these classes are present at the same time? Applying both props `size="large" title="Alert Title"` will apply both classes to the alert message content. Just asking to ensure we don't have any unintentional style overrides if both classes are present at the same time.

Reply:
Yep, that's a valid use case 👍🏻